ibis_substrait/compiler/translate.py

```diff
@@ -932,7 +933,7 @@ def _find_parent_tables(op: ops.Selection) -> set[ops.PhysicalTable]:
     # a schema"
     source_tables = {
         t
-        for t in toposort(op).keys()
+        for t in Graph(op).toposort().keys()  # type: ignore[no-untyped-call]
```

Comment:
The top-level `toposort` function got dropped [here](https://github.com/ibis-project/ibis/pull/7758/files#diff-8516547b54230f69120865c854f04ac1815899eda6a1df2da6e5174b79d9c53aL476). It was delegating to `Graph.toposort` under the hood.

Reply:
Mypy was complaining about the `Graph` constructor being untyped. Is there a better fix than suppressing?
```
ibis_substrait/compiler/translate.py: note: In function "_find_parent_tables":
ibis_substrait/compiler/translate.py:936: error: Call to untyped function "Graph" in typed context  [no-untyped-call]
Found 1 error in 1 file (checked 2 source files)
```